echopype/convert/set_groups_azfp.py

```diff
@@ -209,7 +209,7 @@ def set_vendor(self) -> xr.Dataset:
                                'calendar': 'gregorian',
                                'long_name': 'Timestamp of each ping',
                                'standard_name': 'time',
-                               'units': 'seconds since 1970-01-01'}),
+                               'units': 'seconds since 1900-01-01'}),
```

Comment:
I suggest changing the datetime string to a fully explicit one: 1900-01-01T00:00:00Z